include/boost/json/impl/object.ipp

```diff
@@ -70,6 +70,75 @@ find_in_object<string_view>(
     object const& obj,
     string_view key) noexcept;
 
+template< bool SmallTable, bool IgnoreDuplicates >
+void init_from_unchecked( object& obj, unchecked_object& uo )
+{
+    // insert all elements, keeping
+    // the last of any duplicate keys, unless IgnoreDuplicates is false.
```

Comment:
There should be an assert here, the precondition of the function requires `capacity() >= uo.size()`